cypress/integration/pages/mediaAssetPage/tests.js

```diff
@@ -49,9 +49,9 @@ export const testsThatFollowSmokeTestConfig = ({ service, pageType }) => {
       });
     });
 
-    it('legacy MAP should render a link using an <a> with href rather than a plain text <link>', () => {
+    it.only('legacy MAP should render a link using an <a> with href rather than a plain text <link>', () => {
```

Comment:
is this intentional?

```suggestion
    it('legacy MAP should render a link using an <a> with href rather than a plain text <link>', () => {
```